reportable.gemspec

```diff
@@ -14,7 +14,7 @@ pkg_files += Dir['spec/**/*.{rb,yml,opts}']
 Gem::Specification.new do |s|
 
   s.name    = %q{reportable}
-  s.version = '1.4.2'
+  s.version = '1.5.0'
```

Comment:
I think incrementing the version is unrelated and should be done in a separate PR. Also this would be a patch release as it only fixes a bug and doesn't actually add anything.